include/soci/sqlite3/soci-sqlite3.h

```diff
@@ -345,15 +345,23 @@ struct sqlite3_session_backend : details::session_backend
             case db_double:
                 return "real";
             case db_date:
+                return "date";
             case db_int8:
+                return "int1";
```

Comment:
Why do we need these weird names (I've never seen `int1` used anywhere, have you?) instead of just using `int8`? AFAICS SQLite doesn't care anyhow, as long as the type contains `int`.

Or, even better, why not use the names already recognized by `get_data_type_map()`, i.e. `tinyint` here etc?

Reply:
Will change to the previous names